packages/finch-graphql-devtools/app/scripts/components/MessageViewer/helpers.js

```diff
@@ -0,0 +1,11 @@
+export const formatJSON = json => {
+  return JSON.stringify(json, null, ' ')
+}
+
+export const safeParse = json => {
```

Comment:
out of curiosity, why call it `safeParse` and not something like `parseJSON`?

Reply:
safeParse indicates that when we parse it's going to not throw. JSON.parse throws with bad data so this being `safe` allows use to assume its not going to throw on bad input.